nixos/modules/config/qt5.nix

```diff
@@ -12,7 +12,13 @@ let
   isLxqt = cfg.platformTheme == "lxqt";
   isKde = cfg.platformTheme == "kde";
 
-  packages = if isQGnome then [ pkgs.qgnomeplatform pkgs.adwaita-qt ]
+  packages =
+    if isQGnome then [
+      pkgs.qgnomeplatform
+      pkgs.adwaita-qt
+      pkgs.qgnomeplatform-qt6
+      pkgs.adwaita-qt6
```

Comment:
I am not sure if this is the right thing to do as this module seems to be specific to Qt5 theming.

Reply:
Then maybe rename the module to qt. QT4 apps are removing. I thought most users want to theme all qt apps.